content/en-us/reference/engine/libraries/os.yaml

```diff
@@ -300,10 +300,10 @@ functions:
     code_samples:
   - name: os.clock
     summary: |
-      Returns a high-precision amount of CPU time used by Lua in seconds,
+      Returns a high-precision amount of elapsed CPU time in seconds,
       intended for use in benchmarking.
     description: |
-      Returns the amount of CPU time used by Lua in seconds. This value has high
+      Returns the amount of elapsed CPU time in seconds. This value has high
```

Comment:
I think `amount of elapsed CPU time` a redundant statement. It could probably be shortened to just `elapsed CPU time`